src/foam/nanos/crunch/Capability.js

```diff
@@ -101,6 +101,11 @@ foam.CLASS({
       class: 'StringArray',
       documentation: `List of permissions granted by this capability`
     },
+    {
+      name: 'permissionsIntercepted',
+      class: 'StringArray',
```

Comment:
This works and it's consistent with `permissionsGranted`, but something makes me think that they should both be many-to-many relationships, not string arrays. Not in the scope of this PR though.

Reply:
That makes sense. Right now searching a capability that intercepts a permission requires selecting all capabilities and iterating over every permissionsIntercepted list.

Reply:
Yeah. There are pros and cons to each approach. Groups used to have string arrays of permissions they implied though and they can be annoying to work with in a lot of different ways. This PR is big enough as is though so let's leave this as a string array for now.